src/adapters/Cornerstone/Segmentation_4X.js

```diff
@@ -669,6 +1000,8 @@ function insertPixelDataPlanar(
             }
         }
     }
+
+    return 1;
```

Comment:
why is this here?

Reply:
C++ intensifies.

Reply:
you are right, the return is not needed

Reply:
also https://github.com/dcmjs-org/dcmjs/pull/155/files#diff-077798721d0df020ee8184a2f54d4459dba916e19c86ac0f0aecc4e6ea80bf2fR890 is nto needed. It was something from an old version where I was returning the number of label maps. But now we use the length of the labelmapBufferArray.

Thanks for pointing out